docs/tables/aws_lambda_function_event_source_mapping.md

```diff
@@ -0,0 +1,61 @@
+# Table: aws_lambda_function_event_source_mapping
+
+AWS Lambda is a compute service that lets you run code without provisioning or managing servers.
+Event Source Mappings are triggers that route events from services like Kafka, Kinesis, SQS or SNS to the configured AWS Lambda.
+
+## Examples
+
+### Basic Info
+
+```sql
+select
+  event_source_arn,
+  function_arn,
+  function_name,
+  last_processing_result,
+  parallelization_factor,
+  state,
+  destination_config
+from
+  aws_lambda_function_event_source_mapping;
+```
+
+### List of lambda functions which disabled
+
+```sql
+select
+    function_name, 
+    state, 
+    last_modified, 
+    state_transition_reason
+from
+    aws_lambda_function_event_source_mapping
+where
+    state = 'Disabled';
+```
+
+### Count of lambda function event source mappings by runtime engines of the function
```

Comment:
This example seems more relevant to the `aws_lambda_function` table, are there any other examples we can add specific to event source mappings? For instance, showing the data in `source_access_configurations`? Or showing  event sources with a particular filter criteria (from `filter_criteria`)?